### DISCUSSION TITLE
我是如何用Obsidian进行创作管理的

### DISCUSSION THREAD

**Discussion by edonyzpc** [edony-ink/comments#49] [General]
🚀

**Comment by rebron1900** [comment]
我也是多年的Ghost用户了，期待你的分享。
（目前在OB中写的文章发布到Ghost上如果写了标签会自动转换成一级标题，有点困扰

**Reply by edonyzpc** [reply]
不知道你有没有自己修改 ghost 引擎的代码，我先假设你没有修改 ghost server 标签相关的代码，分享一下自己的想法：
- 你说的标签自动转成一级标题的问题，可能是格式的问题（`# tag` vs. `#tag`)，也可能是发布插件的 bug，这个可以再排查一下；
- 当前 ghost 正文应该不支持类似`#tag` 这样格式的标签的，只在 post setting 可以设置 tags（ghost server 版本是 5.87.1）；
- 如果想在 ghost 中支持类似 obsidian 一样的 tag 的话，可以参考我的另外一片文章:[Ghost Post 中支持标签的方法](https://www.edony.ink/ghost-post-supporting-tags/)，这个方法同样适用于在 obsidian 中设置，这样就可以保证 publish 出来的内容不需要修改了；

**Reply by rebron1900** [reply]
我要的不是这个效果。
我的周刊都是现在OB里写好再复制粘贴到Ghost上的，而在OB中我会给一些周刊中的一些内容加上标签直接用 `#tag` 这种形式，所以复制粘贴到Ghost中后这些标签会被识别成一级标题，这个应该是Ghost所使用的转码库的问题。

**Reply by rebron1900** [reply]
另外我暂时还没使用这个工具用OB发表Ghost文章。
因为担心内容不可控，反正都要去修改，索性复制粘贴，如果格式有问题再直接进行修改。
